package.json

```diff
@@ -81,11 +81,11 @@
     "@testing-library/react-hooks": "3.7.0",
     "@welldone-software/why-did-you-render": "^6.1.4",
     "babel-core": "7.0.0-bridge.0",
+    "babel-runtime": "^6.26.0",
```

Comment:
do we really need this `babel-runtime`?

Reply:
yes 😢 during yarn test. (this was the reason given by test explaining why they failed)